Makefile

```diff
@@ -99,7 +99,6 @@ endif
 # On Windows / MinGW $(CC) is undefined by default.
 ifeq ($(YQ2_OSTYPE),Windows)
 CC := gcc
-CXX := g++
```

Comment:
The Makefile changes should be their own separate commit after everything else.

Reply:
thank you fixed in 6f3c223